src/mock/mock-personal-tasks.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
same for this? I am assuming these are for tests

Reply:
I created this to test the crud.js file, but need to finalize the test file